mkt/purchase/tests/test_webpay.py

```diff
@@ -174,16 +178,27 @@ def jwt_dict(self, expiry=3600, issued_at=None, contrib_uuid=None):
             },
         }
 
-    def jwt(self, req=None, **kw):
+    def jwt(self, req=None, encoding_secret=None, encode_kw=None, **kw):
         if not req:
             req = self.jwt_dict(**kw)
-        return jwt.encode(req, self.webpay_dev_secret)
+        if not encode_kw:
```

Comment:
i like the ternary style of `encode_kw = encode_kw or {}`, since its one line less :)